cmd/dendrite-demo-yggdrasil/yggconn/node.go

```diff
@@ -26,7 +26,7 @@ import (
 	"os"
 	"strings"
 
-	"github.com/lucas-clemente/quic-go"
```

Comment:
I think the problem is it doesn't statically know that `quic-go` is `quic`. So:
```go
quic "github.com/lucas-clemente/quic-go"
```
should fix it.

Reply:
Also tried that before, Goland complained about redundant alias.
`typecheck` is even disabled in the `golangci-lint` config.

Somehow, after reverting that commit, the linter seems to be happy again.